opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -232,61 +284,246 @@ message MetricDescriptor {
     //   12. The 1 second collection cycle ends. A metric is exported for the
     //      number of requests received over the interval of time t_1 to
     //      t_0+1 with a value of 1.
-    CUMULATIVE = 3;
+    //
+    // Note that the first value in a sequence of CUMULATIVE metrics
+    // after a reset is equivalent in value to a DELTA metric for the
+    // period since the reset.  Although a CUMULATIVE metric could
+    // technically be reset after every collection event and still be
+    // described as CUMULATIVE, exporters should use DELTA if there is
+    // no intention of repeating the StartTimeUnixNano timestamp.
+    CUMULATIVE = 0x4;
+
+    // The two Structure kinds are exclusive of one another.  One
+    // of the Structure kind elements MUST be set.
+
+    // ADDING structure means the measurement determines a sum.  For
+    // DELTA kind this is expressed as the change in sum since the
+    // last collection.  For CUMULATIVE kind this is expressed as the
+    // last collected value of the sum.  For INSTANTANEOUS kind this
+    // is expressed as a scalar value that would be added into a Sum.
+    ADDING = 0x8;
+
+    // GROUPING structure means the value has been computed by
+    // combining individual values through a meaningful aggregation.
+    // GROUPING structure implies the sum of measurements is not
+    // necessarily meaningful.  These may be expressed as DELTA
+    // kind, indicating an aggregation of values since the last
+    // collection, or as CUMULATIVE kind, indicating an aggregation
+    // of values since the last StartTimeUnixNano reset.  These may
```

Comment:
Why have a concept of grouping that is not based on the last collection? Why this separate reset option? And why is this the case only for `GROUPING` but not for `DELTA`?